antlr_plsql/plsql.g4

```diff
@@ -4125,6 +4134,7 @@ VARIABLE:                     V A R I A B L E;
 VARRAY:                       V A R R A Y;
 VARRAYS:                      V A R R A Y S;
 VARYING:                      V A R Y I N G;
+VERBOSE:                      V E R B O S E;
```

Comment:
If we are adding it here, we should also add it to the list of keywords that are also accepted an id. (And check if explain and analyze are there already.)